JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+  agent {
+    node {
+      label 'project:any'
+    }
+  }
+  stages {
+    stage('Clean Workspace') {
+      steps {
+        cleanWs()
+      }
+    }
+    stage('Git Clone') {
+      steps {
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']],
+        doGenerateSubmoduleConfigurations: false,
+        extensions: [],
+        submoduleCfg: [],
+        userRemoteConfigs: [[credentialsId: 'CIDA-Jenkins-GitHub',
+        url: 'https://github.com/usgs/aqts-capture-db.git']]])
+      }
+    }
+    stage('Download liquibase jar') {
+      steps {
+        sh '''mkdir $WORKSPACE/rc
+          if [ ! -f rc/liquibase ]; then
+            /usr/local/bin/aws s3 cp s3://owi-common-resources/resources/InstallFiles/liquibase/liquibase-$LIQUIBASE_VERSION.tar.gz $WORKSPACE/rc/liquibase.tar.gz
```

Comment:
Thinking out loud: we specify liquibase version 3.8.5 as part of the travis build - is there a good way to make sure we're using that same version in this liquibase script?

Reply:
We should get errors If the version used is not compatible with the script.. Also limited by what has been put on S3...

Reply:
Right, that makes sense.